scripts/jenkins-yoctobuild-build.sh

```diff
@@ -1147,7 +1148,7 @@ if [ "$PUBLISH_ARTIFACTS" = true ]; then
             version=$($WORKSPACE/integration/extra/release_tool.py --version-of $image --in-integration-version HEAD)
             # Upload binaries.
             case "$image" in
-                deployments*|deviceadm|deviceauth|gui|integration|inventory|mender-api-gateway-docker|mender-conductor*|useradm*)
+                deployments*|deviceadm|deviceauth|gui|integration|inventory|mender-api-gateway-docker|mender-conductor*|tenantadm*|useradm*)
```

Comment:
We could add here `org-welcome-email-preparer` also

Reply:
No, if you look above, this is using `release_tool.py --list git`, and this list doesn't include `org-welcome-email-preparer`. On the `docker` list does.

Reply:
Thanks for clarifying